### ISSUE TITLE
Bug: declare disappear 

### ISSUE THREAD

**Issue by jorge-graca-sky** [timocov/ts-transformer-minify-privates#24]
Hi, thanks for this amazing effort!

I did notice that when using `useDefineForClassFields` on TS config plus `declare` when I add your transformer to the code it removes the `declare`.

Example code:
https://www.typescriptlang.org/play?useDefineForClassFields=true#code/MYGwhgzhAEBCYBdFmgbwLAChrQA4FcAjEAS2GgBMBTUMAJyrwHskEmAuaCBOkgOwDmAbiw5cvAG6JG1Wg2jB6dFiOwKmfbnXzA2dABQBKNKJwKlLaAF5oAIluqcAXywvMWWlGgB5PiQ0magTEZMysHFw8-MKm4iRSCIyKdMoI1nYOrkA

**Comment by timocov** [comment]
> it removes the declare.

@jorge-graca-sky what does it mean? do you have an example of input and output?

**Comment by jorge-graca-sky** [comment]
It means that TS compiles as if there was no `declare` key word written:
```
class A {
  private b: number;
}

class C {
  private declare d: number;
}
```
When I use the TS-transformer-minify-privates both these classes output the same js code with Object.defineProperty in the contructor. Class C should not do so.

**Comment by timocov** [comment]
Fixed in [v0.6.0](https://github.com/timocov/ts-transformer-minify-privates/releases/tag/v0.6.0).
